docs/changelog_3_4_0.rst

```diff
@@ -105,6 +105,7 @@ Miscellaneous
 -------------
 
 - Added JSON schema files for ``info.json`` files (:issue:`4375`)
+- Replaced ``cogs.red`` with new Red Index (:issue:`4439`)
```

Comment:
We actually replaced the link to the approved repo list on CogBoard with Index, `cogs.red`-like service is still planned.